CONTRIBUTING.md

```diff
@@ -39,96 +39,104 @@ Every day at 10:30 we have standups in Zoom meeting: <id>
 Approximately every 2 weeks we have sprint reviews and then also a sprint
 planning meeting.
 
-## Procedure
 
 
-1. Create an issue on Github, and talk to the team members on the NBIS
-   local-ega Slack channel. You can alternatively pick one already
-   created.
+## How to work on a new feauture/bug
 
+Create an issue on Github, and talk to the team members on the NBIS local-ega
+Slack channel. You can alternatively pick one already created.
 
-2. Assign yourself to that issue.
+Assign yourself to that issue.
 
-#. Discussions on how to proceed about that issue take place in the
-   comment section on that issue, beforehand.
+Discussions on how to proceed about that issue take place in the comment
+section on that issue, beforehand.
 
-   The keyword here is *beforehand*. It is usually a good idea to talk
-   about it first. Somebody might have already some pieces in place,
-   we avoid unnecessary work duplication and a waste of time and
-   effort.
+The keyword here is *beforehand*. It is usually a good idea to talk about it
+first. Somebody might have already some pieces in place, we avoid unnecessary
+work duplication and a waste of time and effort.
 
-#. Work on it (on a fork, or on a separate branch) as you wish. That's
-   what `git` is good for. This GitHub repository follows
-   the [coding guidelines from NBIS].
-   
-   Name your branch as you wish and prefix the name with:
+
+## How we work with Git
+
+All work take place in feature branches. Give your branch a short descriptive
+name and prefix the name with:
 
    * `feature/` if it is a code feature
    * `hotfix/` if you are fixing an urgent bug
 
-   Use comments in your code, choose variable and function names that
-   clearly show what you intend to implement.
+Use comments in your code, choose variable and function names that clearly show
+what you intend to implement.
+
+Once the feature is done it is merged back into `master` by making a Pull
+Request.
+
+
+### General stuff about git and commit messages
+
+In general it is better to commit often. Small commits are easier to roll back
+and also makes the code easier to review.
+
+Write helpful commit messages that describes the changes and possibly why they
+were necessary.
+
+Each commit should contain changes that are functionally connected and/or
+related. If you for example want to write _and_ in a commit message this is a
+good sign that it should have been two commits.
+
+Learn how to select chunkgs of changed files to do multiple separate commits of
+unrelated things. This can be done with either `git add -p` or `git commit -p`.
 
-   Use [git rebase -i] in
-   order to rewrite your history, and have meaningful commits.  That
-   way, we avoid the _'Typo'_, _'Work In Progress (WIP)'_, or
-   _'Oops...forgot this or that'_ commits.
 
-   Limit the first line of your git commits to 72 characters or less.
+#### Helpfull commit messages
```

Comment:
I think we should use "helpful" instead of "helpfull". It's not technically incorrect, but "helpful" is the more common spelling.